db/migrate/20180601171211_create_workflow_node.rb

```diff
@@ -0,0 +1,13 @@
+class CreateWorkflowNode < ActiveRecord::Migration[5.0]
+  def change
+    create_table :workflow_nodes do |t|
```

Comment:
`workflow_nodes` is probably too generic for this.  I think the name needs to be scoped to the configuration models.  Maybe even just something like `configuration_workflow_nodes`.

@Fryguy

Reply:
done prefixing the table with `configuration_`